cerebrium/cortex/introduction.mdx

```diff
@@ -67,18 +67,24 @@ cerebrium deploy <MODEL_NAME> <PRIVATE_API_KEY> --hardware=<HARDWARE>
 For the parameters above:
 
 - **\<MODEL_NAME\>**: Give your model a name for you to recognize on your Cerebrium dashboard
-- **\<HARDWARE\>**: This is either CPU, GPU or A10
 - **\<PRIVATE_API_KEY\>**: This is the private API key you can find on your Cerebrium dashboard
+- **\<HARDWARE\>**: This is either CPU, GPU or A10
+
+Additionally, we provide two flags to further control which files you add to your deployment. They are:
+
+- **--include**: Regrex string of files or folders to include. eg. `--include "./modelFiles/" --include my-awesome-file.txt`
+- **--exclude**: The files or folders to exclude. eg. `--exclude "[.venv, ./private.py, ./folderToExclude]"`
```

Comment:
pretty minor but do we need the brackets?

Reply:
if that will take more than 5 mins to change then its safe to ignore 😆

Reply:
It will work without the brackets as is! It's a strip("[]") and then split(",") so if there's no brackets to strip it's not a problem. 
If you think it's clearer to have no brackets then we'll go with that method!